app/components/CarouselCard.tsx

```diff
@@ -51,7 +97,7 @@ type LinkProps = {
   openLink: string | ((event: GestureResponderEvent) => void)
 }
 
-const Link: VFC<LinkProps> = ({ openLink, text }) => {
+const Link: React.FunctionComponent<LinkProps> = ({ openLink, text }) => {
```

Comment:
VFC is deprecated in React 18.

React.FunctionComponent no longer implicitly declares children prop. (we should probably also move off of React.FC and declare children props where necessary)